storage/innobase/trx/trx0trx.cc

```diff
@@ -374,7 +373,6 @@ trx_t *trx_create()
 
 #ifdef WITH_WSREP
 	trx->wsrep_event= NULL;
```

Comment:
Shouldn’t this be checked as part of `trx_t::assert_freed()`? Also the assertions on `trx_t::mod_tables` and `trx_t::lock` belong there.

Reply:
Sure but that is out of scope of this change.